src/components/responsive-flyout/styles/OverlayHeader.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../../styles/variables';
+@import './variables';
+
+.overlay-header {
+    position: relative;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: $overlay-header-height;
+    padding: 13px 16px;
+    background-color: white;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, .08);
```

Comment:
Is there a standard shadow that we can reuse?

Reply:
Not that I've seen that matches design